src/nodedef.cpp

```diff
@@ -890,13 +890,13 @@ void CNodeDefManager::updateTextures(IGameDef *gamedef,
 				for (u32 i = 0; i < 6; i++)
 					tiledef[i].name += std::string("^[noalpha");
 			}
-			if (f->waving == 1)
```

Comment:
this is the allfaces_optional one. I assume that in the future people will start writing "waving = 2" for new leaves, since it matches the documentation, and by forcing it to wave like leaves with >= 1, we avoid that leaves wouldn't weave with waving == 1.